--- swift/utils/build-presets.ini-orig	2021-09-23 08:59:28.810313021 -0500
+++ swift/utils/build-presets.ini	2021-10-01 15:06:55.158508958 -0500
@@ -785,7 +785,6 @@
 libicu
 libcxx
 
-build-ninja
 install-llvm
 install-swift
 install-lldb
@@ -866,6 +865,65 @@
     buildbot_linux
     mixin_buildbot_linux,no_test
 
+# Preset for Fedora Linux
+[preset: buildbot_linux_fedora,no_test]
+mixin-preset=
+    mixin_lightweight_assertions,no-stdlib-asserts
+    mixin_linux_install_components_with_clang
+build-subdir=buildbot_linux
+
+lldb
+release
+foundation
+libdispatch
+indexstore-db
+sourcekit-lsp
+lit-args=-v --time-tests
+
+llbuild
+swiftpm
+swift-driver
+xctest
+libicu
+libcxx
+
+install-foundation
+install-libdispatch
+reconfigure
+
+install-llvm
+install-swift
+install-lldb
+install-llbuild
+install-swiftpm
+install-swift-driver
+install-xctest
+install-libicu
+install-prefix=/usr
+install-libcxx
+install-sourcekit-lsp
+build-swift-static-stdlib
+build-swift-static-sdk-overlay
+build-swift-stdlib-unittest-extra
+
+# Path to the root of the installation filesystem.
+install-destdir=%(install_destdir)s
+
+# Path to the .tar.gz package we would create.
+installable-package=%(installable_package)s
+
+# Executes the lit tests for the installable package that is created
+# Assumes the swift-integration-tests repo is checked out
+
+test-installable-package
+
+# This ensures the default module cache
+# location is local to this run, allowing
+# to schedule multiple builds safely
+# in Linux CI bots
+relocate-xdg-cache-home-under-build-subdir
+
+
 [preset: buildbot_linux_crosscompile_android,tools=RA,stdlib=RD,build]
 mixin-preset=
     mixin_lightweight_assertions,no-stdlib-asserts
