--- swift-corelibs-foundation/Sources/Foundation/Port.swift.orig	2024-07-10 20:15:01.454636074 -0500
+++ swift-corelibs-foundation/Sources/Foundation/Port.swift	2024-07-10 20:23:56.859897469 -0500
@@ -92,9 +92,11 @@
 
 #if canImport(Glibc) && !os(Android) && !os(OpenBSD)
 import Glibc
-fileprivate let SOCK_STREAM = Int32(Glibc.SOCK_STREAM.rawValue)
-fileprivate let SOCK_DGRAM  = Int32(Glibc.SOCK_DGRAM.rawValue)
-fileprivate let IPPROTO_TCP = Int32(Glibc.IPPROTO_TCP)
+// https://forums.swift.org/t/cannot-build-toolchain-on-ubuntu-24-04-lts/72200
+import CoreFoundation
+fileprivate let SOCK_STREAM = Int32(CoreFoundation.SOCK_STREAM.rawValue)
+fileprivate let SOCK_DGRAM  = Int32(CoreFoundation.SOCK_DGRAM.rawValue)
+fileprivate let IPPROTO_TCP = Int32(CoreFoundation.IPPROTO_TCP)
 #endif
 
 #if canImport(Glibc) && os(Android) || os(OpenBSD)
