--- swift/lib/Driver/ToolChains.cpp.orig	2018-11-16 09:22:38.058780122 -0600
+++ swift/lib/Driver/ToolChains.cpp	2018-11-16 09:31:08.699326699 -0600
@@ -881,7 +881,7 @@
     useLLDB = true;
     break;
   case REPLJobAction::Mode::PreferLLDB:
-    useLLDB = !findProgramRelativeToSwift("lldb").empty();
+    useLLDB = true;
     break;
   }
 
@@ -909,7 +909,7 @@
   ArgStringList Arguments;
   Arguments.push_back(context.Args.MakeArgString(std::move(SingleArg)));
 
-  return {"lldb", Arguments};
+  return {"/usr/libexec/swift-lldb/lldb", Arguments};
 }
 
 ToolChain::InvocationInfo
