--- swiftpm/Sources/Build/BuildDescription/ClangModuleBuildDescription.swift.orig	2024-11-23 08:51:29.212470347 -0600
+++ swiftpm/Sources/Build/BuildDescription/ClangModuleBuildDescription.swift	2024-11-23 08:51:57.609260467 -0600
@@ -406,7 +406,7 @@
         case .debug:
             return ["-O0"]
         case .release:
-            return ["-O2"]
+            return ["-O1"]
         }
     }
 
