--- lldb/source/Plugins/ExpressionParser/Clang/ClangHost.cpp.orig	2019-06-25 14:34:32.837621023 -0500
+++ lldb/source/Plugins/ExpressionParser/Clang/ClangHost.cpp	2019-06-26 16:03:32.304254695 -0500
@@ -54,11 +54,11 @@
   static const llvm::StringRef kResourceDirSuffixes[] = {
       // LLVM.org's build of LLDB uses the clang resource directory placed
       // in $install_dir/lib{,64}/clang/$clang_version.
-      "lib" CLANG_LIBDIR_SUFFIX "/clang/" CLANG_VERSION_STRING,
+      "/swift" "/clang/" CLANG_VERSION_STRING,
       // swift-lldb uses the clang resource directory copied from swift, which
       // by default is placed in $install_dir/lib{,64}/lldb/clang. LLDB places
       // it there, so we use LLDB_LIBDIR_SUFFIX.
-      "lib" LLDB_LIBDIR_SUFFIX "/lldb/clang",
+      "/swift" "/clang",
   };
 
   for (const auto &Suffix : kResourceDirSuffixes) {
