--- compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_posix.cc.orig	2019-10-30 12:10:54.591060832 -0500
+++ compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_posix.cc	2019-10-30 12:11:56.483063580 -0500
@@ -1131,7 +1131,7 @@
 CHECK_SIZE_AND_OFFSET(ipc_perm, cgid);
 #if !defined(__aarch64__) || !SANITIZER_LINUX || __GLIBC_PREREQ (2, 21)
 /* On aarch64 glibc 2.20 and earlier provided incorrect mode field.  */
-CHECK_SIZE_AND_OFFSET(ipc_perm, mode);
+/* CHECK_SIZE_AND_OFFSET(ipc_perm, mode); */ /* Error with clang 9.0.0 */
 #endif
 
 CHECK_TYPE_SIZE(shmid_ds);
