--- llvm-project/compiler-rt/lib/builtins/CMakeLists.txt.orig	2019-12-11 15:39:19.537518740 -0600
+++ llvm-project/compiler-rt/lib/builtins/CMakeLists.txt	2019-12-11 23:23:57.583292289 -0600
@@ -537,7 +537,7 @@
   add_subdirectory(macho_embedded)
   darwin_add_builtin_libraries(${BUILTIN_SUPPORTED_OS})
 else ()
-  set(BUILTIN_CFLAGS "")
+  set(BUILTIN_CFLAGS "-D_DEFAULT_SOURCE")
 
   append_list_if(COMPILER_RT_HAS_STD_C11_FLAG -std=c11 BUILTIN_CFLAGS)
 
