--- llvm-project/compiler-rt/lib/builtins/CMakeLists.txt.orig	2019-12-11 15:39:19.537518740 -0600
+++ llvm-project/compiler-rt/lib/builtins/CMakeLists.txt	2019-12-11 15:37:50.815021079 -0600
@@ -539,7 +539,7 @@
 else ()
   set(BUILTIN_CFLAGS "")
 
-  append_list_if(COMPILER_RT_HAS_STD_C11_FLAG -std=c11 BUILTIN_CFLAGS)
+  append_list_if(COMPILER_RT_HAS_STD_C11_FLAG -std=c11 -D_DEFAULT_SOURCE BUILTIN_CFLAGS)
 
   # These flags would normally be added to CMAKE_C_FLAGS by the llvm
   # cmake step. Add them manually if this is a standalone build.
