--- swift-corelibs-foundation/Sources/CoreFoundation/internalInclude/CoreFoundation_Prefix.h.orig	2024-07-31 10:29:42.005997906 -0500
+++ swift-corelibs-foundation/Sources/CoreFoundation/internalInclude/CoreFoundation_Prefix.h	2024-07-31 10:31:06.147568761 -0500
@@ -200,6 +200,7 @@
 #endif
 
 #if !TARGET_OS_MAC
+#if 0
 #if !HAVE_STRLCPY
 CF_INLINE size_t
 strlcpy(char * dst, const char * src, size_t maxlen) {
@@ -229,6 +230,7 @@
     return dstlen + srclen;
 }
 #endif
+#endif // XYZZY - We do have them in Fedora
 #endif // !TARGET_OS_MAC
 
 #if TARGET_OS_WIN32
