diff -Naur ./indexstore-db-orig/include/IndexStoreDB/Database/IDCode.h indexstore-db/include/IndexStoreDB/Database/IDCode.h
--- ./indexstore-db-orig/include/IndexStoreDB/Database/IDCode.h	2019-08-19 13:15:44.000000000 -0500
+++ indexstore-db/include/IndexStoreDB/Database/IDCode.h	2020-01-23 16:20:05.055701686 -0600
@@ -14,6 +14,7 @@
 #define INDEXSTOREDB_SKDATABASE_IDCODE_H
 
 #include <functional>
+#include <cstdint>
 
 namespace IndexStoreDB {
 namespace db {
