--- lldb/scripts/Python/finishSwigPythonLLDB.py.orig	2019-09-09 10:19:48.545053827 -0500
+++ lldb/scripts/Python/finishSwigPythonLLDB.py	2019-09-09 10:28:59.121109016 -0500
@@ -673,8 +673,16 @@
 
     from distutils.sysconfig import get_python_lib
     if platform.system() == 'Linux':
-        if platform.linux_distribution()[0] == 'Fedora':
-            dbg.dump_text("Platform is Fedora Linux")
+        linux_distro = ''
+        # platform.linux_distribution() is not available in
+        # Python >= 3.8
+        if sys.version_info[0] >= 3 and sys.version_info[1] >= 8:
+            from distro import linux_distribution
+            linux_distro = linux_distribution()[0]
+        else:
+            linux_distro = platform.linux_distribution()[0]
+        if linux_distro == 'Fedora' or linux_distro == 'Red Hat Enterprise Linux' or linux_distro == 'CentOS Linux':
+            dbg.dump_text("Platform is Fedora Linux or Red Hat Enterprise Linux or CentOS Linux")
             # On Fedora the installation gets split into lib and lib64,
             # which prevents building lldb from completing successfully.
             return get_python_lib(False, False, vstrPythonInstallDir)
