diff -Naur llvm-project-orig/clang-tools-extra/clangd/Shutdown.cpp llvm-project/clang-tools-extra/clangd/Shutdown.cpp
--- llvm-project-orig/clang-tools-extra/clangd/Shutdown.cpp	2020-04-20 16:06:02.000000000 -0500
+++ llvm-project/clang-tools-extra/clangd/Shutdown.cpp	2020-04-23 12:35:36.247095797 -0500
@@ -10,6 +10,7 @@
 
 #include <atomic>
 #include <thread>
+#include <cstdlib>
 
 namespace clang {
 namespace clangd {
