diff --color -Naur llbuild-orig/lib/llvm/Support/CMakeLists.txt llbuild/lib/llvm/Support/CMakeLists.txt
--- llbuild-orig/lib/llvm/Support/CMakeLists.txt	2024-09-05 18:18:16.000000000 -0500
+++ llbuild/lib/llvm/Support/CMakeLists.txt	2024-11-20 16:07:54.309067679 -0600
@@ -1,6 +1,8 @@
 # Disable implicit-fallthrough, if used.
 append_if(CXX_SUPPORTS_IMPLICIT_FALLTHROUGH_FLAG "-Wno-implicit-fallthrough" CMAKE_CXX_FLAGS)
 
+SET_SOURCE_FILES_PROPERTIES(Host.cpp PROPERTIES COMPILE_FLAGS -O1)
+
 add_llbuild_library(llvmSupport STATIC
 APFloat.cpp
 APInt.cpp
diff --color -Naur llbuild-orig/utils/import-llvm/cmake/supportCMakeLists.txt llbuild/utils/import-llvm/cmake/supportCMakeLists.txt
--- llbuild-orig/utils/import-llvm/cmake/supportCMakeLists.txt	2024-09-05 18:18:16.000000000 -0500
+++ llbuild/utils/import-llvm/cmake/supportCMakeLists.txt	2024-11-20 16:09:00.316243756 -0600
@@ -1,6 +1,8 @@
 # Disable implicit-fallthrough, if used.
 append_if(CXX_SUPPORTS_IMPLICIT_FALLTHROUGH_FLAG "-Wno-implicit-fallthrough" CMAKE_CXX_FLAGS)
 
+SET_SOURCE_FILES_PROPERTIES(Host.cpp PROPERTIES COMPILE_FLAGS -O1)
+
 add_llbuild_library(llvmSupport
 APFloat.cpp
 APInt.cpp
