diff --color -Naur swift-source-orig/llvm-project/clang/utils/creduce-clang-crash.py swift-source/llvm-project/clang/utils/creduce-clang-crash.py
--- swift-source-orig/llvm-project/clang/utils/creduce-clang-crash.py	2024-06-13 19:54:42.303016942 -0500
+++ swift-source/llvm-project/clang/utils/creduce-clang-crash.py	2024-06-13 19:57:15.922473813 -0500
@@ -15,7 +15,6 @@
 import stat
 import sys
 import subprocess
-import pipes
 import shlex
 import tempfile
 import shutil
@@ -61,7 +60,7 @@
 
 
 def quote_cmd(cmd):
-    return " ".join(pipes.quote(arg) for arg in cmd)
+    return " ".join(shlex.quote(arg) for arg in cmd)
 
 
 def write_to_script(text, filename):
@@ -220,7 +219,7 @@
         )
 
         for msg in self.expected_output:
-            output += "grep -F %s t.log || exit 1\n" % pipes.quote(msg)
+            output += "grep -F %s t.log || exit 1\n" % shlex.quote(msg)
 
         write_to_script(output, self.testfile)
         self.check_interestingness()
diff --color -Naur swift-source-orig/llvm-project/libcxx/utils/libcxx/test/dsl.py swift-source/llvm-project/libcxx/utils/libcxx/test/dsl.py
--- swift-source-orig/llvm-project/libcxx/utils/libcxx/test/dsl.py	2024-06-13 19:55:23.627829920 -0500
+++ swift-source/llvm-project/libcxx/utils/libcxx/test/dsl.py	2024-06-13 19:57:42.124257373 -0500
@@ -8,7 +8,7 @@
 
 import os
 import pickle
-import pipes
+import shlex
 import platform
 import re
 import shutil
@@ -281,7 +281,7 @@
       }
     #endif
   """
-    return programSucceeds(config, program, args=[pipes.quote(l) for l in locales])
+    return programSucceeds(config, program, args=[shlex.quote(l) for l in locales])
 
 
 @_memoizeExpensiveOperation(lambda c, flags="": (c.substitutions, c.environment, flags))
diff --color -Naur swift-source-orig/llvm-project/libcxx/utils/libcxx/test/format.py swift-source/llvm-project/libcxx/utils/libcxx/test/format.py
--- swift-source-orig/llvm-project/libcxx/utils/libcxx/test/format.py	2024-06-13 19:55:23.627829920 -0500
+++ swift-source/llvm-project/libcxx/utils/libcxx/test/format.py	2024-06-13 19:58:01.911604299 -0500
@@ -11,7 +11,7 @@
 import lit
 import lit.formats
 import os
-import pipes
+import shlex
 import re
 import shutil
 
diff --color -Naur swift-source-orig/ninja/configure.py swift-source/ninja/configure.py
--- swift-source-orig/ninja/configure.py	2024-06-13 19:54:39.050795565 -0500
+++ swift-source/ninja/configure.py	2024-06-13 19:56:39.970026531 -0500
@@ -23,7 +23,7 @@
 
 from optparse import OptionParser
 import os
-import pipes
+import shlex
 import string
 import subprocess
 import sys
@@ -264,7 +264,7 @@
 env_keys = set(['CXX', 'AR', 'CFLAGS', 'CXXFLAGS', 'LDFLAGS'])
 configure_env = dict((k, os.environ[k]) for k in os.environ if k in env_keys)
 if configure_env:
-    config_str = ' '.join([k + '=' + pipes.quote(configure_env[k])
+    config_str = ' '.join([k + '=' + shlex.quote(configure_env[k])
                            for k in configure_env])
     n.variable('configure_env', config_str + '$ ')
 n.newline()
