--- swift/utils/build-presets.ini.orig	2024-11-11 12:30:06.743565294 -0600
+++ swift/utils/build-presets.ini	2024-11-11 12:31:15.936936014 -0600
@@ -870,7 +870,7 @@
 # Executes the lit tests for the installable package that is created
 # Assumes the swift-integration-tests repo is checked out
 
-test-installable-package
+# test-installable-package
 
 # Build the benchmarks against the toolchain.
 toolchain-benchmarks
