diff -Naur ./swift-orig/cmake/modules/AddSwift.cmake ./swift/cmake/modules/AddSwift.cmake
--- ./swift-orig/cmake/modules/AddSwift.cmake	2020-10-28 15:44:53.000000000 -0500
+++ ./swift/cmake/modules/AddSwift.cmake	2020-10-31 08:57:43.088514376 -0500
@@ -356,7 +356,7 @@
   if("${CFLAGS_SDK}" STREQUAL "LINUX")
     if(${CFLAGS_ARCH} STREQUAL x86_64)
       # this is the minimum architecture that supports 16 byte CAS, which is necessary to avoid a dependency to libatomic
-      list(APPEND result "-march=core2")
+      list(APPEND result "-mcx16")
     endif()
   endif()
 
diff -Naur ./swift-orig/stdlib/cmake/modules/AddSwiftStdlib.cmake ./swift/stdlib/cmake/modules/AddSwiftStdlib.cmake
--- ./swift-orig/stdlib/cmake/modules/AddSwiftStdlib.cmake	2020-10-28 15:44:53.000000000 -0500
+++ ./swift/stdlib/cmake/modules/AddSwiftStdlib.cmake	2020-10-31 08:58:34.788036005 -0500
@@ -287,7 +287,7 @@
   if("${CFLAGS_SDK}" STREQUAL "LINUX")
     if(${CFLAGS_ARCH} STREQUAL x86_64)
       # this is the minimum architecture that supports 16 byte CAS, which is necessary to avoid a dependency to libatomic
-      list(APPEND result "-march=core2")
+      list(APPEND result "-mcx16")
     endif()
   endif()
 
