--- llvm-project/lldb/source/Plugins/ScriptInterpreter/Python/ScriptInterpreterPython.cpp.orig	2024-08-01 20:37:45.248357585 -0500
+++ llvm-project/lldb/source/Plugins/ScriptInterpreter/Python/ScriptInterpreterPython.cpp	2024-08-01 20:40:07.870815929 -0500
@@ -182,6 +182,7 @@
       return;
 #endif
 
+/*
     if (PyEval_ThreadsInitialized()) {
       Log *log = GetLog(LLDBLog::Script);
 
@@ -195,6 +196,7 @@
     // InitThreads acquires the GIL if it hasn't been called before.
     PyEval_InitThreads();
   }
+*/
 
   PyGILState_STATE m_gil_state = PyGILState_UNLOCKED;
   bool m_was_already_initialized = false;
