--- llvm-project/lldb/source/Plugins/ScriptInterpreter/Python/ScriptInterpreterPython.cpp.orig	2024-08-01 20:37:45.248357585 -0500
+++ llvm-project/lldb/source/Plugins/ScriptInterpreter/Python/ScriptInterpreterPython.cpp	2024-08-03 09:31:36.741687679 -0500
@@ -182,7 +182,8 @@
       return;
 #endif
 
-    if (PyEval_ThreadsInitialized()) {
+    Py_Initialize();
+    if (Py_IsInitialized()) {
       Log *log = GetLog(LLDBLog::Script);
 
       m_was_already_initialized = true;
@@ -191,9 +192,7 @@
                 m_gil_state == PyGILState_UNLOCKED ? "un" : "");
       return;
     }
-
-    // InitThreads acquires the GIL if it hasn't been called before.
-    PyEval_InitThreads();
+   
   }
 
   PyGILState_STATE m_gil_state = PyGILState_UNLOCKED;
