--- sourcekit-lsp/Utilities/build-script-helper.py.orig	2019-10-29 09:41:43.052157614 -0500
+++ sourcekit-lsp/Utilities/build-script-helper.py	2019-10-30 21:23:18.538560646 -0500
@@ -15,7 +15,7 @@
   subprocess.check_call(cmd, env=env)
 
 def swiftpm_bin_path(swift_exec, swiftpm_args, env=None):
-  swiftpm_args = filter(lambda arg: arg != '-v' and arg != '--verbose', swiftpm_args)
+  swiftpm_args = list(filter(lambda arg: arg != '-v' and arg != '--verbose', swiftpm_args))
   cmd = [swift_exec, 'build', '--show-bin-path'] + swiftpm_args
   print(' '.join(cmd))
   return subprocess.check_output(cmd, env=env).strip()
@@ -56,7 +56,7 @@
     install_binary(exe, swiftpm_bin_path, toolchain_bin, toolchain)
 
 def install_binary(exe, source_dir, install_dir, toolchain):
-  cmd = ['rsync', '-a', os.path.join(source_dir, exe), install_dir]
+  cmd = ['rsync', '-a', os.path.join(source_dir.decode('UTF-8'), exe), install_dir]
   print(' '.join(cmd))
   subprocess.check_call(cmd)
 
