diff -Naur swift-orig/utils/build-presets.ini swift/utils/build-presets.ini
--- swift-orig/utils/build-presets.ini	2020-04-21 15:42:23.000000000 -0500
+++ swift/utils/build-presets.ini	2020-04-23 09:55:05.038603369 -0500
@@ -789,7 +789,6 @@
 
 dash-dash
 
-build-ninja
 install-swift
 install-lldb
 install-llbuild
@@ -801,12 +800,7 @@
 install-sourcekit-lsp
 build-swift-static-stdlib
 build-swift-static-sdk-overlay
-build-swift-stdlib-unittest-extra
 
-# Executes the lit tests for the installable package that is created
-# Assumes the swift-integration-tests repo is checked out
-
-test-installable-package
 
 # Build the benchmarks against the toolchain.
 toolchain-benchmarks
