diff --git a/llvm-project/llvm/utils/benchmark/src/benchmark_register.cc b/llvm-project/llvm/utils/benchmark/src/benchmark_register.cc
index dc6f93568..06f9e5e35 100644
--- a/llvm-project/llvm/utils/benchmark/src/benchmark_register.cc
+++ b/llvm-project/llvm/utils/benchmark/src/benchmark_register.cc
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <limits>
+
 #include "benchmark_register.h"
 
 #ifndef BENCHMARK_OS_WINDOWS
