--- swift-docc/build-script-helper.py.orig	2022-01-18 12:34:07.014018324 -0600
+++ swift-docc/build-script-helper.py	2022-01-18 12:34:35.002731959 -0600
@@ -261,7 +261,7 @@
   if verbose:
     print(' '.join([escape_cmd_arg(arg) for arg in cmd]))
   return os.path.join(
-    subprocess.check_output(cmd, env=env).strip(), 'docc')
+    subprocess.check_output(cmd, env=env).strip().decode(), 'docc')
 
 def create_intermediate_directories(dir_path, verbose):
   cmd = ["mkdir", "-p", dir_path]
