diff -Naur swift-orig/utils/build-presets.ini swift/utils/build-presets.ini
--- swift-orig/utils/build-presets.ini	2022-01-16 20:01:17.939502268 -0600
+++ swift/utils/build-presets.ini	2022-01-16 20:02:58.528781444 -0600
@@ -834,7 +834,6 @@
 install-prefix=/usr
 install-libcxx
 install-sourcekit-lsp
-install-swiftdocc
 build-swift-static-stdlib
 build-swift-static-sdk-overlay
 build-swift-stdlib-unittest-extra
