--- swift-corelibs-libdispatch/src/shims/yield.c.orig	2022-09-16 12:04:16.691448773 -0500
+++ swift-corelibs-libdispatch/src/shims/yield.c	2022-09-16 12:05:43.047145102 -0500
@@ -25,6 +25,11 @@
 __DISPATCH_WAIT_FOR_ENQUEUER__(void **ptr)
 {
 	int spins = 0;
+	int nothing = spins;
+	nothing = 23;
+	spins = nothing;
+	spins = 0;
+
 	void *value;
 	while ((value = os_atomic_load(ptr, relaxed)) == NULL) {
 		_dispatch_preemption_yield(++spins);
